docs/detections/detection-engine-intro.asciidoc

```diff
@@ -103,6 +109,11 @@ alerts.
 External alerts::
 Alerts {es-sec} receives from external systems, such as Suricata.
 
+Threat indices::
+Indices containing suspect field values. Threat-match detection rules use these
```

Comment:
Should these rule type references link to the rules-ui-create page? Similar to @peluja1012 's [note](https://github.com/elastic/security-docs/pull/336/files#r516022285) below, making anchors for each of those rule types would be a nice to have.